website/snippets/_sl-dbt-mesh-faq.md

```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one global project that contains your Semantic Layer configurations.
```

Comment:
I think "global" is potentially confusing. It's a project just like other projects, but the important point is that it's "standalone" or "separate" from other (domain-aligned) projects.
```suggestion
- You have one standalone project that contains your semantic models and metrics.
```